Makefile

```diff
@@ -86,4 +86,11 @@ dirtycheck:
 		echo "  >  No uncommitted changes to generated files found."; \
 	fi
 
+lintinsights:
+	@echo "  >  Linting security-insights.yml ..."
+	@curl -O --silent https://raw.githubusercontent.com/ossf/security-insights-spec/refs/tags/v2.0.0/schema.cue
+	@cue vet security-insights.yml schema.cue
+	@rm schema.cue
+	@echo "  >  Linting security-insights.yml complete."
+
```

Comment:
this could be the world's simplest GitHub Action, if we wanted to enable SI vetting for any/everyone 🤔 someday

Reply:
+1 to a GitHub Action. I would be interested in using that!